components/org.wso2.carbon.identity.cors.valve/src/main/java/org/wso2/carbon/identity/cors/valve/internal/handler/CORSRequestHandler.java

```diff
@@ -59,36 +55,20 @@ private CORSManager getCORSManager() {
      * @throws CORSException
      * @throws CORSManagementServiceException
      */
-    public void handleActualRequest(HttpServletRequest request, HttpServletResponse response)
+    public void handleSimpleRequest(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Why have we changed a public method signature?

Reply:
Reverted.